specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2023-09-01-preview/apimgatewayConfigConnections.json

```diff
@@ -0,0 +1,350 @@
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/ConfigConnectionIdParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/IfMatchRequiredParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "The gateway config connection delete request was accepted.",
+            "headers": {
+              "location": {
+                "description": "Location header",
+                "type": "string"
+              }
+            }
+          },
+          "204": {
+            "description": "The gateway config connection does not exist."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    }
+  },
+  "definitions": {
+    "GatewayConfigConnectionBaseProperties": {
+      "type": "object",
+      "properties": {
+        "provisioningState": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The current provisioning state of the API Management gateway config connection ",
+          "enum": [
+            "Activating",
+            "Active",
+            "Created",
+            "Canceled",
+            "Failed",
+            "Stopped",
+            "Succeeded",
+            "Terminating",
+            "Upgrading",
+            "Updating"
+          ],
+          "x-ms-enum": {
+            "name": "ProvisioningState",
+            "modelAsString": true
+          }
+        },
+        "sourceId": {
+          "type": "string",
+          "format": "arm-id",
+          "x-ms-arm-id-details": {
+            "allowedResources": [
+              {
+                "type": "Microsoft.ApiManagement/service/workspaces"
+              }
+            ]
+          },
+          "description": "The link to the API Management service workspace."
+        },
+        "defaultHostname": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The default hostname of the data-plane gateway."
+        },
+        "hostnames": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "The hostnames of the data-plane gateway to which requests can be sent."
+        }
+      }
+    },
+    "ApiManagementGatewayConfigConnectionProperties": {
+      "type": "object",
+      "properties": {},
```

Comment:
>"properties": {}, [](http://example.com/codeflow?start=6&length=17)

You should not have an empty property bag here. The ApiManagementGatewayConfigConnectionResource already contains everything within a property bag.

Reply:
This is same format parent entity has it so I am trying to follow the same pattern

Reply:
It's because the parent is an ARM resource and is required to place RP-specific properties into a property bag. This type is not an ARM resource, just some property definitions already declared within the parent's property bag. This definition doesn't need its own property bag, especially an empty one.